tests/test_connections.py

```diff
@@ -1,7 +1,6 @@
-import urllib.request
 import pytest
+from run import app
 
-SITE_LOCATION = "http://localhost:5000"
 
-def test_connection():
-    assert urllib.request.urlopen(SITE_LOCATION).getcode() == 200
+def test_webserver_running(client):
+    assert client.get('/').status_code == 200
```

Comment:
This test checks only the landing page, do you think we should check all of the available routes?

Reply:
Sure, it's just for start, but surely we want to check out the rest of the routes as well 👍